Samples/Xbox/InteractivitySample.cpp

```diff
@@ -621,6 +648,11 @@ void Sample::ProcessInteractiveEvents(std::vector<interactive_event> events)
         case interactive_event_type::error:
             HandleInteractivityError(event);
             break;
+        case interactive_event_type::unknown:
+            std::shared_ptr<interactive_message_event_args> messageEventArgs = std::dynamic_pointer_cast<interactive_message_event_args>(event.event_args());
+            m_console->Write(messageEventArgs->message().c_str());
+            m_console->Write(L"\n");

```

Comment:
Needs curly braces. Also, another nit, still may want this in a separate handler, so we have clean, consistent sample code (even if it's just a tiny little method)